Azure/sonic-buildimage
# [Mellanox] Update SDK 4.4.1622, FW xx.2008.1622

Signed-off-by: Volodymyr Samotiy <volodymyrs@nvidia.com>



**- Why I did it**
To update SDK/FW to newer versions:  SDK 4.4.1622, FW xx.2008.1622.

SDK:
- Added new Tunnel Decap Rule Dump example Python script. New Tunnel Decap Rule Iterator API sx_api_tunnel_decap_rules_iter_get is introduced. 
- Added GA-level support for SN3700 systems.
- Added new APIs for controlling SFP reset and low power mode
- Bug fix | Port | SN3800 Increased TX FIR amplitude to improve eye margin 
- Bug fix | WJH | Core dumped when disabling RoCE drop group severity after setting drop reason severity. Added support to enable/disable per severity for WJH RoCE drop reason group. 
- Bug fix | STP Failover Latency on 1GbaseT module is longer than expected (up to 250 msec) when using Spectrum-based systems. 
- Bug fix | Split mode with one PLL module (MFS1S50-Vxxx) is not functional in SN3700C systems. 
- Bug fix | SN3800 system will raise 100GbE link with No-FEC by default when the module is LR (instead of RS-FEC). 
- Bug fix | BGP | Spectrum-2 and Spectrum-3 based systems may run into a condition where the SDK process consumes very high memory when running BGP with Max Scale.
- Bug fix | Port toggle | On rare occasion, when connecting DR4 to Mellanox Spectrum-3 system and DR1 to Spectrum system after toggling form both sides, link may not rise. 
- Bug fix | Port toggle | On rare occasion, when connecting DR4 to Mellanox Spectrum-3 systems on both sides, toggling form both sides may cause the link to not rise. 
- Bug fix | Port toggle | Spectrum-3 when partner port is toggled, the linkdown counter increases twice instead of once.  
- Bug fix | Cables | On rare occasion, when connecting between an SN4600C system and ConnectX-4 card with Molex Splitter Copper cable 2x50GbE in AutoNeg with NO-FEC mode, high BER may be observed. 
- Bug fix | Cables , Speed | In SN4700, the link up time using optics cables for 100/200/400GbE may take up to 30 seconds. 

**- How I did it**
Updated appropriate sub-modules and versions.

**- How to verify it**
Build an image and verify all is up and running.

**- Which release branch to backport (provide reason below if selected)**




**- Description for the changelog**

[Mellanox] Update SDK 4.4.1622, FW xx.2008.1622
